### DISCUSSION TITLE
Setting the default theme on factory reset

### DISCUSSION THREAD

**Discussion by bensuffolk** [HASwitchPlate/openHASP#403] [General]
I would like to have the default theme set to Material Dark instead of HASP Dark. Is there a build option I can specify in user_config_override.h to make this happen?

**Comment by fvanroie** [comment]
Not yet, but it's very easy so I added `HASP_START_DIM`, `HASP_START_PAGE`,  `HASP_THEME_ID` in commit 95b28eb222e0c8bbcdac29a00bb5a59c9419336e.
